workflow-examples/jobs-in-parallel/jobs_in_parallel.groovy

```diff
@@ -0,0 +1,10 @@
+def branches = [:]
+for (int i = 0; i < 4; i++) {
+  def which = "${i}" 
```

Comment:
Ok, but as Jesse says there, you should make it clear that `dummy` is an actual parameter that is defined in `test_jobs`, right?

In any case, this comes back to the topic of adding enough documentation to this example :)